backends/platform/sdl/morphos/morphos.cpp

```diff
@@ -41,7 +56,7 @@ void OSystem_MorphOS::init() {
 }
 
 bool OSystem_MorphOS::hasFeature(Feature f) {
-	if (f == kFeatureOpenUrl)
+	if (f == kFeatureOpenUrl || f == kFeatureClipboardSupport)
```

Comment:
This can be removed since both `kFeatureOpenUrl` and `kFeatureClipboardSupport` are handled in `OSystem_SDL::hasFeature(f);`.